tests/test_stac_core.py

```
@@ -0,0 +1,62 @@
+import pytest
+import stac_mjx
+from dm_control import mjcf
```

Ruff (0.8.2)

`dm_control.mjcf` imported but unused

Remove unused import: `dm_control.mjcf`

(F401)